pkgdb/src/resolver/manifest-raw.cc

```diff
@@ -528,10 +531,11 @@ to_json( nlohmann::json & jto, const HookRaw & hook )
 void
 HookRaw::check() const
 {
-  if ( this->script.has_value() && this->file.has_value() )
+  if ( this->script.has_value() && this->onActivate.has_value() )
     {
       throw InvalidManifestFileException(
-        "hook may only define one of 'hook.script' or 'hook.file' fields." );
+        "hook may only define one of 'hook.script' or `hook.on-activate` "
+        "fields." );
     }
```

Comment:
I dont think this is still true.
But if it is its not documented in the manpage afaict

Reply:
It is still true, it's just left out of the `manifest.toml` man page.

Reply:
Fixed